opal/mca/accelerator/null/accelerator_null_component.c

```diff
@@ -31,7 +31,7 @@ const char *opal_accelerator_null_component_version_string
 static int accelerator_null_open(void);
 static int accelerator_null_close(void);
 static int accelerator_null_component_register(void);
-static opal_accelerator_base_module_t* accelerator_null_init(int *priority);
```

Comment:
This commit should have occurred before the last one, no reason to add something and then change it immediately in the next commit

Reply:
Moved the component selection commit before the NULL Accelerator Commit